osf/models/metaschema.py

```diff
@@ -53,6 +56,23 @@ def get_latest_versions(self, only_active=True):
             latest_schemas = latest_schemas.filter(active=True)
         return latest_schemas.order_by('name', '-schema_version').distinct('name')
 
+    def get_latest_versions_and_allow_egap_admins(self, request, only_active=True):
```

Comment:
Why is this code split off of `get_latest_versions` as opposed to being incorporated into the existing function?

Reply:
I figured this is should be removed once EGAP is complete, so I didn't want change anything too much. I'd rather add something obviously EGAP, then remove it, then change another innocent by-standard function.

Reply:
That's fair. It could be a little more elegant to call a subset of the code in `get_latest_versions_and_allow_egap_admins` as a function in `get_latest_versions` so that the function call in `api.schemas.views` stays constant and removing the egap special case is as easy as removing the function call and deleting the function code. But generally, this makes sense and feel free to keep it the same if you think it's better this way.